app/controllers/feedback_forms_controller.rb

```diff
@@ -22,6 +33,10 @@ def create
 
   protected
 
+  def set_form_type
+    @form_type = params.permit(:type)[:type]
+  end
+
   def url_regex
     /.*href=.*|.*url=.*|.*http:\/\/.*|.*https:\/\/.*/i
```

Comment:
can you just parse the URL with `URI.parse` to see whether it's valid?

Reply:
Great idea. I'm not sure of the historical reasoning behind this, but if its ok, I'm going to leave as is since this PR isn't touching that yet.